WooCommerce/Classes/ViewRelated/Search/SearchViewController.swift

```diff
@@ -362,14 +362,14 @@ extension SearchViewController: SyncingCoordinatorDelegate {
     /// Synchronizes the models for the Default Store (if any).
     ///
     func sync(pageNumber: Int, pageSize: Int, reason: String?, onCompletion: ((Bool) -> Void)? = nil) {
-        let keyword = self.keyword
+        let keyword = searchUICommand.sanitizeKeyword(self.keyword)
         searchUICommand.synchronizeModels(siteID: storeID,
                                           keyword: keyword,
                                           pageNumber: pageNumber,
                                           pageSize: pageSize,
-                                        onCompletion: { [weak self] isCompleted in
+                                          onCompletion: { [weak self] isCompleted in
                                             // Disregard OPs that don't really match the latest keyword
-                                            if keyword == self?.keyword {
+                                            if keyword == self?.searchUICommand.sanitizeKeyword(self?.keyword ?? String()) {
```

Comment:
It may be good to do a `guard let self = self else { return }` to avoid so many optional chaining